lib/ffi.ts

```diff
@@ -1,4 +1,7 @@
 import { join } from "https://deno.land/std@0.170.0/path/mod.ts";
+// can be use to improve error handling but requires extra --allow-read permission
+// import { existsSync } from "https://deno.land/std@0.170.0/fs/mod.ts";
```

Comment:
Remove these lines: I'll improve the error handling by checking if the error messages... Eventually :)